machine_configuration/mco-update-boot-images.adoc

```diff
@@ -16,11 +16,13 @@ This process could cause the following issues:
 * Certificate expiration issues
 * Version skew issues
 
-To avoid these issues, you can configure your cluster to update the boot image whenever you update your cluster. By modifying the `MachineConfiguration` object, you can enable this feature. Currently, the ability to update the boot image is available for only Google Cloud Platform (GCP) clusters and is not supported for clusters managed by the Cluster API.
+To avoid these issues, you can configure your cluster to update the boot image whenever you update your cluster. By modifying the `MachineConfiguration` object, you can enable this feature. Currently, the ability to update the boot image is available for only Google Cloud Platform (GCP) clusters and as a Technology Preview feature for Amazon Web Services (AWS) clusters. It is not supported for clusters managed by the {cluster-capi-operator}.
 
-:FeatureName: The updating boot image feature
+:FeatureName: The updating boot image feature for AWS
 include::snippets/technology-preview.adoc[]
 
+If you are not using the default user data secret, named `worker-user-data`, in your machine set, or you have modified the `worker-user-data` secret, you should not use managed boot image updates. The Machine Config Operator (MCO) requires the default user data secret in order to update the boot image on your nodes. 
```

Comment:
Minor rephrase:
```suggestion
If you are not using the default user data secret, named `worker-user-data`, in your machine set, or you have modified the `worker-user-data` secret, you should not use managed boot image updates. This is because the Machine Config Operator (MCO) will update the MachineSet to use a managed version of the secret. By using the managed boot images feature, you are giving up the capability to customize the secret stored in the MachineSet object.
```
The feature does not _require_ the default user data secret. It just means that by turning on the feature, any user customization done to the secret will not be used in the MachineSet in the future.